requirements.txt

```diff
@@ -4,7 +4,8 @@ GitPython>=3.1.18
 joj-submitter>=0.0.8
 loguru>=0.5.3
 mattermostdriver>=7.3.2
+pandas>=2.0.0
```

Comment:
`pandas` is too heavy for the project with lots of dependencies. It should be enough to just use `csv` package.